src/layout/GetStarted.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import {
+  Button,
+  Flex,
+  FlexItem,
+  PageSection,
+  PageSectionVariants,
+  Title,
+} from '@patternfly/react-core';
+import './GetStarted.scss';
+import { getStarted as getStarted } from '../consts';
+
+const GetStarted = () => (
+  <PageSection
+    className="ins-c-get-started ins-m-display-md pf-m-center"
+    landing-page-type="unauthenticated"
+    isWidthLimited
+    variant={PageSectionVariants.light}
+  >
+    <Flex
+      direction={{ default: 'column' }}
+      spaceItems={{ default: 'spaceItemsLg' }}
+      alignItems={{ default: 'alignItemsCenter' }}
+    >
+      <FlexItem>
+        <Title
+          headingLevel="h2"
+          size="4xl"
+          className="ins-c-section-title ins-m-align-center"
+        >
+          {getStarted.title}
+        </Title>
+      </FlexItem>
+      <FlexItem>
+        <Button
+          className="ins-m-marketing"
+          variant="primary"
+          isLarge
```

Comment:
Is this the main login button? If so, can we add an ouiaId prop to this one? QE can use it to easily locate the button within the DOM. 

It is enough to just add a prop like
```
ouiaId="primary-login-button"
```

For the top right login button, it'd be nice if we had some ouiaId like
```
ouiaId="top-right-login-button"
```

Reply:
@john-dupuy top right one is coming from chrome

Reply:
oh yeah! let me go add an id to that one

Reply:
Thanks!